openedx/core/djangoapps/schedules/utils.py

```diff
@@ -15,3 +20,24 @@ def __init__(self, *args, **kwargs):
 
     def log_debug(self, message, *args, **kwargs):
         LOG.debug(self.log_prefix + ': ' + message, *args, **kwargs)
+
+
+def get_value_from_settings(name, site=None, site_config_name=None):
+    if site_config_name is None:
+        site_config_name = name
+
+    if site is None:
+        site = get_current_site()
+
+    site_configuration = None
+    if site is not None:
+        try:
+            site_configuration = getattr(site, "configuration", None)
+        except SiteConfiguration.DoesNotExist:
+            pass
+
+    value_from_settings = getattr(settings, name, None)
+    if site_configuration is not None:
+        return site_configuration.get_value(site_config_name, default=value_from_settings)
+    else:
+        return value_from_settings
```

Comment:
Why do we have to do this one-off just for schedules?  Shouldn't this logic be used throughout the edx-platform?

Reply:
Probably? I moved this to the theming helpers, does that seem like a good place to put it?